lib/static/components/state/state-error.jsx

```diff
@@ -46,6 +56,24 @@ class StateError extends Component {
         return null;
     }
 
+    _tryToImproveSnippet(snippet) {
+        const filePrefix = "| // ";
+        const filePrefixBeginIndex = snippet.indexOf(filePrefix);
+        const filePrefixEndIndex = snippet.indexOf("\x1B", filePrefixBeginIndex + filePrefix.length);
+
+        if (filePrefixBeginIndex === -1 || filePrefixEndIndex === -1) {
+            return snippet;
+        }
```

Comment:
We are trying to find it by assuming our first line is "| // <file-name>" ended with control symbol (to stop painting text with gray, in this case)

Looks fragile, but i dont know any other option